addons/pos_hr/models/pos_order.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from functools import partial
+
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    employee_id = fields.Many2one(
```

Comment:
employee_id = fields.Many2one( 'hr.employee',string='Employee',states={'done': [('readonly', True)], 'invoiced': [ ('readonly', True)]},  help="Person who uses the cash register. It can be a reliever, a student or an interim employee.",)